src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 410px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+    &__image-container {
+      margin: 32px 19px 0;
+    }
+
+    &__image {
+      width: 160px;
+      height: 134px;
+      flex-shrink: 0;
+    }
+
+    &__name {
+      margin: 40px 16px 4px;
+      color: $color-main;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &__code {
+      color: $color-secondary;
+      margin-left: 16px;
+      font-size: 10px;
+      font-weight: 400;
```

Comment:
it's a default browser value. so no needs to redeclare this rule. Delete for the same cases as well
```suggestion
```

Reply:
not fixed